client/lxd.go

```diff
@@ -221,7 +221,7 @@ func lxdParseResponse(resp *http.Response) (*api.Response, string, error) {
 
 	// Handle errors
 	if response.Type == api.ErrorResponse {
-		return nil, "", api.StatusErrorf(resp.StatusCode, response.Error)
+		return nil, "", api.StatusErrorf(resp.StatusCode, "%s", response.Error)
```

Comment:
@markylaing added api.NewStatusError for this scenario to avoid needing the parsing cost of fmt.Errorf